code/modules/antagonists/clockcult/clock_mobs/_eminence.dm

```diff
@@ -189,12 +189,12 @@
 			command_text = "The Eminence orders that defenses should be built starting from the top of Reebe!"
 	if(marker_icon)
 		new/obj/effect/temp_visual/ratvar/command_point(get_turf(A), marker_icon)
-		for(var/mob/M in servants_and_ghosts())
+		for(var/mob/M in SSticker.mode.servants_of_ratvar)
 			to_chat(M, "<span class='large_brass'>[replacetext(command_text, "GETDIR", dir2text(get_dir(M, command_location)))]</span>")
```

Comment:
this should probably be seen by ghosts in some manner

Reply:
I don't think it's necessary, but I can readd it for this, but just remove the sound for ghosts